examples/webgl_loader_gltf_compressed.html

```diff
@@ -0,0 +1,143 @@
+			Toy Car by Guido Odendahl<br />
+			<a href="https://hdrihaven.com/hdri/?h=royal_esplanade" target="_blank" rel="noopener">Royal Esplanade</a> by <a href="https://hdrihaven.com/" target="_blank" rel="noopener">HDRI Haven</a>
+		</div>
+
+		<script src="./js/libs/basis/msc_basis_transcoder.js"></script>
+
+		<script type="module">
+
+			import * as THREE from '../build/three.module.js';
+
+			import { OrbitControls } from './jsm/controls/OrbitControls.js';
+			import { GLTFLoader } from './jsm/loaders/GLTFLoader.js';
+			import { RGBELoader } from './jsm/loaders/RGBELoader.js';
+			import { RoughnessMipmapper } from './jsm/utils/RoughnessMipmapper.js';
+
+			import { KTX2Loader } from './jsm/loaders/KTX2Loader.js';
+			import { MeshoptDecoder } from './jsm/libs/meshopt_decoder.module.js';
+
+			let camera, scene, renderer;
+
+			init();
+			render();
+
+			function init() {
+
+				const container = document.createElement( 'div' );
+				document.body.appendChild( container );
+
+				renderer = new THREE.WebGLRenderer( { antialias: true } );
+				renderer.setPixelRatio( window.devicePixelRatio );
+				renderer.setSize( window.innerWidth, window.innerHeight );
+				renderer.toneMapping = THREE.ACESFilmicToneMapping;
+				renderer.toneMappingExposure = 1;
+				renderer.outputEncoding = THREE.sRGBEncoding;
+				container.appendChild( renderer.domElement );
+
+				camera = new THREE.PerspectiveCamera( 45, window.innerWidth / window.innerHeight, 0.025, 2 );
+				camera.position.set( - 1.8, 0.6, 2.7 );
+
+				scene = new THREE.Scene();
+
+				const ktx2Loader = new KTX2Loader().detectSupport( renderer );
+
+				new RGBELoader()
+					.setDataType( THREE.UnsignedByteType )
+					.setPath( 'textures/equirectangular/' )
+					.load( 'royal_esplanade_1k.hdr', function ( texture ) {
+
+						const envMap = pmremGenerator.fromEquirectangular( texture ).texture;
+
+						scene.background = envMap;
+						scene.environment = envMap;
+
+						texture.dispose();
+						pmremGenerator.dispose();
+
+						render();
+
+						// model
+
+						// use of RoughnessMipmapper is optional
+						const roughnessMipmapper = new RoughnessMipmapper( renderer );
+
+						const loader = new GLTFLoader().setPath( 'models/gltf/' );
+						loader.setKTX2Loader( ktx2Loader );
+						loader.setMeshoptDecoder( MeshoptDecoder );
+
+						// ToyCar.glb was produced from the ToyCar scene in glTF-Sample-Models using gltfpack:
+						// gltfpack -i ~/glTF-Sample-Models/2.0/ToyCar/glTF/ToyCar.gltf -o ToyCar.glb -cc -tc -tu -tq 1
+						// The resulting model uses EXT_meshopt_compression (for geometry) and KHR_texture_basisu (for texture compression using UASTC+zstd)
+
+						loader.load( 'ToyCar.glb', function ( gltf ) {
+
+							gltf.scene.traverse( function ( child ) {
+
+								if ( child.isMesh ) {
+
+									// TOFIX RoughnessMipmapper seems to be broken with WebGL 2.0
+									// roughnessMipmapper.generateMipmaps( child.material );
```

Comment:
Probably best to omit RoughnessMipmapper, I'm not sure when this will be fixed.

Reply:
Good call especially since even if it's fixed it's not going to be compatible with compressed textures anyway! Fixed.